gensrc/thrift/HeartbeatService.thrift

```diff
@@ -22,7 +22,7 @@ const i64 IS_SET_DEFAULT_ROWSET_TO_BETA_BIT = 0x01;
 
 struct TMasterInfo {
     1: required Types.TNetworkAddress network_address
-    2: required Types.TClusterId cluster_id
+    2: required Types.TClusterId cluster_id     //deprecated
```

Comment:
You can change it into optional in this change, and delete it in the later version.